styleguide

```diff
@@ -244,12 +244,63 @@ module Styleguide
     end
   end
 
+  class Simplecov < Thor::Group
+    include Helpers
+
+    class_option :defaults, default: true, type: :boolean, desc: 'Whether to import the default configuration'
```

Comment:
Because you have an old config? You don't like them? Want something simple? All generators have this option.